### ISSUE TITLE
Node 16 + 18 ESM import errors

### ISSUE THREAD

**Issue by agoldis** [currents-dev/cypress-cloud#95]
### Before opening, please confirm:

- [X] I have searched for [duplicate or closed issues](https://github.com/currents-dev/cypress-cloud/issues) and [discussions](https://github.com/currents-dev/cypress-cloud/discussions).
- [X] I have done my best to include a minimal, self-contained set of instructions for consistently reproducing the issue.
- [X] I acknowledge that I will attach a **full debug log**, otherwise the issue will be closed with no response.

### Environment information

<details>

```
# Put output below this line


```

</details>

### Describe the bug

@lpanger

On node 16.13.1 I get:
TypeError [ERR_UNKNOWN_FILE_EXTENSION]: Unknown file extension ".json" for /Users/xxxx/node_modules/cypress/package.json

Is node 17+ a requirement?

On node 18.15.0
TypeError [ERR_IMPORT_ASSERTION_TYPE_MISSING]: Module "file:///Users/xxxxx/node_modules/cypress/package.json" needs an import assertion of type "json"

https://github.com/lpanger/cypress-cloud-module-api

cypress-cloud 1.4.5-beta.1
cypress 12.8.1

clone repo and run node index.js

### Expected behavior

Importing ESM modules should not throw on major node versions

### Command and Setup

N/A

### Full log and debug output

<details>

```
// Put your logs below this line


```

</details>

**Comment by agoldis** [comment]
@lpanger

you can use cjs imports as a workaround

**Comment by agoldis** [comment]
@lpanger 
The issue should be resolved in 1.4.5-beta.2

```
npm i cypress-cloud@1.4.5-beta.2
```


Please note, https://github.com/lpanger/cypress-cloud-module-api/blob/main/index-import-file.js is not supposed to work because no such [entry point](https://nodejs.org/api/packages.html#package-entry-points) exists.

Please confirm it's working for you, I'll cut a latest release

**Comment by agoldis** [comment]
Awesome! Releases 1.5.0
